docarray/documents/image.py

```diff
@@ -92,10 +92,10 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[ImageUrl]
-    tensor: Optional[ImageTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[ImageBytes]
+    url: Optional[ImageUrl] = None
```

Comment:
also this?

Reply:
same as above, you will see this at a lot of place in the codebase